locales/en/messages.json

```diff
@@ -822,6 +822,15 @@
     "initialSetupButtonResetZaxisValue": {
         "message": "Reset Z axis, offset: $1 deg"
     },
+    "initialSetupHeading": {
+        "message": "Heading:"
+    },
+    "initialSetupPitch": {
+        "message": "Pitch:"
+    },
+    "initialSetupRoll": {
+        "message": "Roll:"
+    },
```

Comment:
I always recommend to add a description to let know the translators where the strings are used.